cib/remote.c

```diff
@@ -324,7 +324,9 @@ cib_remote_listen(gpointer data)
 
     crm_client_init();
     new_client = calloc(1, sizeof(crm_client_t));
+    CRM_ASSERT(new_client != NULL);
     new_client->remote = calloc(1, sizeof(crm_remote_t));
```

Comment:
There is a big difference between being undefined in the C standard and undefined in the various implementations.
At the point `gcc` or `clvm` change their behaviour, I will agree to sprinkling the code with asserts.

Until then, we have other fish to fry :)